vaadin-crud-flow-parent/vaadin-crud-flow/src/main/java/com/vaadin/flow/component/crud/Crud.java

```diff
@@ -190,25 +190,20 @@ public Crud() {
         deleteButton.getElement().setAttribute("slot", "delete-button");
         deleteButton.addThemeNames("tertiary", "error");
         getElement().appendChild(deleteButton.getElement());
-
-        addAttachListener(e -> {
-            getElement()
-                    .executeJs("this.__validate = function () {return true;}");
```

Comment:
This call was here before the changes introduced by the feature branch. Should we keep it there?

Reply:
True that. Restored.